k8s/deployment.yaml

```
@@ -0,0 +1,39 @@
+# k8s/deployment.yml
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)